osf/models/institution.py

```diff
@@ -8,19 +8,43 @@
 
 
 class Institution(Loggable, base.ObjectIDMixin, base.BaseModel):
+
     # TODO Remove null=True for things that shouldn't be nullable
-    banner_name = models.CharField(max_length=255, null=True, blank=True)
+    # e.g. CharFields should never be null=True
+
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True)
```

Comment:
I think I made this change due to an outdated review:

![outdated-review](https://cloud.githubusercontent.com/assets/3750414/25101058/c8e5fd6c-2380-11e7-90f6-844a1eb369ed.png)